WordPressAuthenticator/Unified Auth/StoredCredentialsAuthenticator.swift

```diff
@@ -120,6 +124,18 @@ class StoredCredentialsAuthenticator: NSObject {
 @available(iOS 13, *)
 extension StoredCredentialsAuthenticator: LoginFacadeDelegate {
     func needsMultifactorCode() {
+        guard let loginFields = loginFields else {
+            return
+        }
+        
+        guard let vc = Login2FAViewController.instantiate(from: .login) else {
```

Comment:
This should use the new UI. The new VC is called TwoFAViewController.

Reply:
Good observation.  I didn't realize this was the old VC.

I also updated the expected tracked events in the WPiOS PR description.